ipatests/prci_definitions/nightly_rawhide.yaml

```diff
@@ -33,11 +33,191 @@ jobs:
       args:
         git_repo: '{git_repo}'
         git_refspec: '{git_refspec}'
-        template: &ci-master-frawhide
-          name: freeipa/ci-master-frawhide
+        template: &ci-master-rawhide
+          name: freeipa/ci-master-rawhide
```

Comment:
Why change `frawhide` to `rawhide`? The only available template is `freeipa/ci-master-frawhide`: https://app.vagrantup.com/freeipa/boxes/ci-master-frawhide

Reply:
Was a typo, rebased thanks!